lib/src/working_copy.rs

```diff
@@ -145,7 +145,7 @@ pub enum SnapshotError {
         /// This path itself is valid UTF-8.
         path: PathBuf,
         /// The symlink target with invalid UTF-8.
-        target: PathBuf,
+        target: Vec<u8>,
```

Comment:
nit: might be better to simply drop the `target` field (or use lossy string.)

Unix path can be converted to bytes, but if we add support for real Windows symlinks, things might get complicated.

Reply:
Makes sense. Done.